plugins/Transitions/lang/en.json

```diff
@@ -19,6 +19,7 @@
         "FromPreviousSiteSearchesInline": "%s from internal searches",
         "FromSearchEngines": "From search engines",
         "FromSocialNetworks": "From social networks",
+        "FromAIAssistants": "From AI assistants",
```

Comment:
We should give the capitalization of "AI (A|a)ssistant" a final check in the placeholder PR.

I think `Referrers_TypeAIAssistants` could very well need a lowercase `a` to match the other entries, but so will some (and not all) of the translations in other open PRs), but it is imho too annoying to look at every single translation in each PR individually 🙈